packages/jellyfish-api-core/__tests__/category/blockchain/getMempoolInfo.test.ts

```diff
@@ -21,10 +21,10 @@ describe('getMempoolInfo', () => {
     const mempoolInfo: MempoolInfo = await client.blockchain.getMempoolInfo()
 
     expect(typeof mempoolInfo.loaded).toStrictEqual('boolean')
-    expect(mempoolInfo.size instanceof BigNumber).toStrictEqual(true)
-    expect(mempoolInfo.bytes instanceof BigNumber).toStrictEqual(true)
-    expect(mempoolInfo.usage instanceof BigNumber).toStrictEqual(true)
-    expect(mempoolInfo.maxmempool instanceof BigNumber).toStrictEqual(true)
+    expect(typeof mempoolInfo.size).toStrictEqual('number')
+    expect(typeof mempoolInfo.bytes).toStrictEqual('number')
+    expect(typeof mempoolInfo.usage).toStrictEqual('number')
+    expect(typeof mempoolInfo.maxmempool).toStrictEqual('number')
```

Comment:
glad you take ownership on this
does that not BigNumber needed??

https://github.com/DeFiCh/ain/blob/8e6cdc7a4a681066c5959ffbea2bc065722f1857/src/rpc/blockchain.cpp#L1498-L1504